code/modules/admin/verbs/adminhelp.dm

```diff
@@ -820,7 +882,7 @@ GLOBAL_DATUM_INIT(ahelp_tickets, /datum/admin_help_tickets, new)
 	var/list/adm = get_admin_counts()
 	var/list/activemins = adm["present"]
 	. = length(activemins)
-	if(. <= 0)
+	if(.)
```

Comment:
```suggestion
	if(!.)
```
Ok, you'll have to review why you made the change in the first place, but unless you are doing an early return the proc should follow the original logic here: if there are no `activemins`, then determine if they are stealthed, AFK or powerless, or if there truly are no admins online.